docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+* **Item max lock time** - When a User edits a Campaign, Email, Landing Page, etc., Mautic locks the item to prevent simultaneous edits by other Users. When the initial User saves and closes or cancels out, the item may remain locked for this period of time. The default is ``0 seconds``.
+
+* **Translate page titles** - To translate page titles from non-Latin (non-English) characters to Latin characters (English).
+
+
 Update settings
 ===============
 
+.. image:: images/update-settings.png
+  :width: 600
+  :alt: Screenshot showing Update Settings Configuration in Mautic
+
+* **Set the minimum stability level required for updates** - This allows you to receive notifications for early access releases. Always read the release notes before updating to an early access release. Set the minimum stability level required for updates. 
+
+* **Update Mautic through Composer [BETA]**  - Set to 'Yes' if you update Mautic through Composer. This is a prerequisite if you want to install and update Plugins through the Marketplace. This becomes the default installation and update method in Mautic 5.
+
 Theme settings
 **************
 
+.. image:: images/theme-settings.png
+  :width: 600
+  :alt: Screenshot showing Theme Settings Configuration in Mautic
+
+* **Default Theme** - Applies a Theme to any Form which doesn’t have a Theme already applied. If you don’t have a Landing Page for your preference center, but have preference settings turned on in Email settings, Mautic creates a default preference center page using the Form styling from the Theme selected here.
+
 API settings
 ************
 
+.. image:: images/api-settings.png
+  :width: 600
+  :alt: Screenshot showing API Settings Configuration in Mautic
+
+Full API documentation is available here.
+
+* **API enabled** - Select Yes to pass data in and out of Mautic through the API.
+
+* **Enable http basic auth?** - Enables basic authentication for Mautic’s API. Mautic recommends only using with secure sites (http).
+
+* **Access token lifetime** - When authorizing a new application or Integration, limits how long the access token is valid (in minutes). The default is ``60`` minutes.
+
+For example - You add a new Integration to your SaaS platform. Enter 30 here to limit the access token validity to ``30`` minutes. If you haven’t completed the authentication in that period of time, you must revalidate.
+
+* **Refresh token lifetime** - When using OAuth 2.0, the lifetime of the refresh token used to request a new access token once expired. Once the refresh token expires, you must reauthorize. The default is ``14`` days.
+
 Asset settings
 **************
 
+.. image:: images/assets-settings.png
+  :width: 600
+  :alt: Screenshot showing Assets Settings Configuration in Mautic
+
+* **Path to the Asset directory** - Set the absolute path to where Assets is uploaded. In order to prevent the public from accessing Assets, use a directory outside of the public web root.
```

Comment:
```suggestion
* **Path to the Asset directory** - Set the absolute path to where Assets are uploaded. In order to prevent the public from accessing Assets, use a directory outside of the public web root.
```